packages/dappmanager/src/domains.ts

```diff
@@ -1,53 +1,75 @@
 import { params } from "@dappnode/params";
 import { getContainerDomain } from "@dappnode/types";
 
-export function stripCharacters(s: string): string {
-  return s.replace(RegExp("_", "g"), "");
-}
-
-export function stripBadDomainChars(s: string): string {
-  // eslint-disable-next-line no-useless-escape
-  return s.replace(/[^a-zA-Z\-]+/g, "-");
-}
+export type ContainerNames = { serviceName: string; dnpName: string };
```

Comment:
Is this being used without `isMainOrMonoservice` somewhere? You might want to add it to this type

Reply:
It is being used without it, we'd better leave this